packages/botonic-react/src/utils.js

```diff
@@ -126,3 +126,15 @@ export const getWebchatElement = () =>
 export const setWebchatElementHeight = newHeight => {
   getWebchatElement().style.height = newHeight
 }
+
+export const serializeRegexs = (_, value) => {
```

Comment:
serializeRegexs => serializeRegex

Since this needs to be called from stringify, I suggest
1) either you add a comment "to be used as 2s argument of JSON.stringify"
2) or we could replace it with a function which already calls JSON.Srtingify passing it as the 2n argument